lib/diaspora_federation/validators/relayable_validator.rb

```diff
@@ -0,0 +1,11 @@
+module DiasporaFederation
+  module Validators
+    module RelayableValidator
+      def self.included(model)
+        model.class_eval do
+          rule :parent_guid, :guid
```

Comment:
The only applied rule was "not_empty", which actually isn't true. There are [no restriction](https://github.com/diaspora/diaspora/blob/develop/db/schema.rb#L170-L171) in DB schema and ```parent_author_signature``` is actually nil in [some](https://github.com/diaspora/diaspora/blob/develop/lib/diaspora/relayable.rb#L106-L109) circumstances.